### DISCUSSION TITLE
Sim Dashboard

### DISCUSSION THREAD

**Discussion by G3ry0** [jason0611/FS22_DashboardLive#9] [Ideas]
This mod is really cool! Will it ever be available in the sim dashboard app? It would be best if you could manage the settings with a tablet and put together a very cool dashboard. I would love to see it included in this app at some point. Of course, I am aware that you are not making this application.

**Comment by jason0611** [comment]
Sim Dashboard is a stand-alone-app, which I can't address with Dashboard Live. So there will be no possible integration, i'm afraid.
